plugins/utils/ccc.nix

```diff
@@ -0,0 +1,44 @@
+{
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "ccc";
+  originalName = "ccc.nvim";
+  defaultPackage = pkgs.vimPlugins.ccc-nvim;
+
+  maintainers = [helpers.maintainers.JanKremer];
+
+  settingsOptions = {
+    default_color = helpers.defaultNullOpts.mkString "#000000" ''
+      The default color used when a color cannot be picked. It must be HEX format.
+    '';
+
+    highlight_mode = helpers.defaultNullOpts.mkString "bg" ''
+      Option to highlight text foreground or background. It is used to output_line and highlighter.
+      Options: "fg" | "bg" | "foreground" | "background"
+    '';
+
+    lsp = helpers.defaultNullOpts.mkBool true ''
+      Whether to enable nvim-lsp support. The color information is updated in the background and the result is used by |:CccPick| and highlighter.
+    '';
+
+    highlighter.auto_enable = helpers.defaultNullOpts.mkBool false ''
+      Whether to enable automatically on BufEnter.
+    '';
+
+    highlighter.lsp = helpers.defaultNullOpts.mkBool true ''
```

Comment:
Could you add both attributes in a single attr set (highlighter  ={ ... };). This is more in line with the rest of the plugins?